### PR TITLE
Change AES-CTS modes CS2 and CS3 to also be inside the fips module.

### PR DESCRIPTION
**Pull request by slontis**
The initial thought was that only CS1 mode (the NIST variant) was allowed.
The lab has asked if these other modes should be included.
The algorithm form indicates that these are able to be validated.

##### Checklist

- [ ] documentation is added or updated
- [x] tests are added or updated

### CODE REVIEW

File: test/recipes/30-test_evp.t [hunk 1]
```diff
@@ -41,6 +41,7 @@ my @files = qw(
                 evpciph_aes_ccm_cavs.txt
                 evpciph_aes_common.txt
                 evpciph_aes_cts1.txt
+                evpciph_aes_cts23.txt
```
Comment (paulidale) [inline test/recipes/30-test_evp.t:44]: Is it worthwhile merging these two files together and just have `evpciph_aes_cts.txt`?

Comment (slontis) [reply test/recipes/30-test_evp.t]: Done

**Review by paulidale — COMMENTED** [review]
👍

**Review by t8m — APPROVED** [review]

**Comment by slontis** [comment]
Thanks. Merged to master.

**Pull request closed without merging**